analyses/molecular-subtyping-integrate/01-integrate-subtyping.Rmd

```diff
@@ -203,6 +203,59 @@ histology<- histology %>%
 
 ```
 
+#### Write out table for `harmonized_diagnosis` -> `cancer_group` mapping
+
+To make it easy for people to examine the mapping between these two fields, we'll write out a table to `results/`.
+We'll only include a subset of the mappings in the README (those that we think require the most explanation), so we'll save a separate table for that purpose.
+
+```{r}
+# We'll save this table in order to have a table of the mappings between the 
+# harmonized dx and cancer group. It will include additional columns that 
+# explain whether or not the subtype is stripped and whether not additional 
+# modifications beyond that have been performed.
+mapping_table <- histology %>% 
+  # Consider removing this sample? -- seems to be a special case, see chunk above
+  # dplyr::filter(sample_id != "7316-2577") %>%
+  dplyr::select(harmonized_diagnosis, 
+                cancer_group) %>%
+  dplyr::distinct() %>%
+  dplyr::filter(!is.na(harmonized_diagnosis)) %>%
+  # This next step removes the subtypes from harmonized diagnosis
+  dplyr::mutate(cancer_group_pre_mod = str_extract(harmonized_diagnosis, 
```

Comment:
I would rename this to `harmonized_diagnosis_nosubtype` or similar. I find this column name `cancer_group_pre_mod` confusing without a lot of context.
Edit - I see later it is renamed to `stripped_subtype`. I might just start with this name.